src/PaymentMethods/Bancontact/Bancontact.php

```diff
@@ -72,6 +75,7 @@ public function payOneClick(): TransactionResponse
 
     /**
      * @return TransactionResponse
+     * @deprecated deprecated since version 1.7.0, please use authorize method
      */
     public function authenticate(): TransactionResponse
     {
```

Comment:
why not do $this->authorize(); so we don't have duplicate code

Reply:
Good point